packages/bitcore-wallet-service/src/lib/model/txproposal.ts

```diff
@@ -178,7 +181,7 @@ export class TxProposal {
     x.setInputs(opts.inputs);
     x.fee = opts.fee;
 
-    if (x.version === 4) {
+    if (x.version === 3) {
```

Comment:
I dont think this is right, lockUntilBlockHeight should only be used on version 4